src/components/ContentEditor/ContentEditor.module.scss

```diff
@@ -1,51 +1,41 @@
 @use '~styles/global.scss';
 @use '~styles/breakpoints.scss';
 
-.content {
-  padding: 2.4rem 1.6rem;
-}
-
-.textInput {
-  display: grid;
-  grid-template-columns: 5fr 1fr;
-}
-
 .textInput textarea {
```

Comment:
Não estilize tags diretamente no CSS. Os estilos devem ser agnósticos ao HTML escrito. Se algum dia algum dev, quiser que isso seja um input ou content editor, isso não deveria quebrar o seu estilo.

Também evite especificar tags encadeadas. Isso é ruim para performance do website como um todo, pois o navegador lê as tags da direita para esquerda, então ele busca TODOS os textareas do APP para então depois filtrar quais estão dentro do .textInput 

Tente manter o css o mais plano possível, sem muito encadeamento.

você pode ler mais sobre escalabilidade de css aqui: 
https://frontstuff.io/you-need-to-stop-targeting-tags-in-css